cron/config/config.yaml

```diff
@@ -21,3 +21,6 @@ bigquery-table: scorecard
 shard-size: 10
 webhook-url: 
 metric-exporter: stackdriver
+# UPGRADEv2: to remove.
```

Comment:
One more thing you'll need is to add `BUCKET_V2` env var [here](https://github.com/ossf/scorecard/blob/main/cron/k8s/controller.release.yaml#L60) and [here](https://github.com/ossf/scorecard/blob/main/cron/k8s/worker.release.yaml#L35). That way, the release tests will write to a different bucket and not to prod bucket.

Reply:
got it, thanks. I completely overlooked the release tests :-(

Reply:
ptal, updated